finagle-core/src/main/scala/com/twitter/finagle/util/LoadService.scala

```diff
@@ -21,10 +21,14 @@ import scala.io.Source
  */
 private object ClassPath {
 
-  private val ignoredPackages = Seq(
+  private val defaultIgnoredPackages = Seq(
     "apple/", "ch/epfl/", "com/apple/", "com/oracle/",
     "com/sun/", "java/", "javax/", "scala/", "sun/", "sunw/")
 
+  private[util] def ignoredPackages =
```

Comment:
Yeah, my guess is that it's very inexpensive.  If it starts being a bottleneck, we can work on it.